lib/PCAP/Bwa/Meta.pm

```diff
@@ -37,7 +37,7 @@ use YAML qw(LoadFile);
 
 use PCAP::Bam;
 
-const my @INIT_KEYS => qw(in temp fastq paired_fq cram bam);
+const my @INIT_KEYS => qw(in temp fastq paired_fq illumina_fq cram bam);
```

Comment:
This new `illumina_fq` flag is set per input object, and tracks whether the `_R[12]_001` file names are used, or the `_[12]` file names.